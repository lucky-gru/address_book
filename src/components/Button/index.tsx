import styled from 'styled-components'

const Button = styled.span`
  text-transform: uppercase;
  padding: 8px 20px;
  height: 42px;
  line-height: 42px;
  border-radius: 21px !important;
  text-align: center;
  background: #ff4c50;
  border-color: #ff4c50;
  color: #fff;
  font-weight: 500;
  cursor: pointer;
`

export default Button;
