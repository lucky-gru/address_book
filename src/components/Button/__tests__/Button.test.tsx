import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Button from '../index'

test('Button render', () => {
  const tree = renderer.create(<Button />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('background', '#ff4c50')
  expect(tree).toHaveStyleRule('color', '#fff')
})
