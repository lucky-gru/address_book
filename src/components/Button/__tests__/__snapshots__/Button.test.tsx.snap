// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render 1`] = `
.c0 {
  text-transform: uppercase;
  padding: 8px 20px;
  height: 42px;
  line-height: 42px;
  border-radius: 21px !important;
  text-align: center;
  background: #ff4c50;
  border-color: #ff4c50;
  color: #fff;
  font-weight: 500;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c0:hover {
  background: #bb0f13;
}

<span
  className="c0"
/>
`;
